rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -272,6 +272,9 @@ private void writeRDTDetailsToWidgets(WidgetArgs widgetArgs, String deviceId, De
                 CovidConstants.FormFields.DETECTED_COMPONENT_TYPE,
                 detectedComponentType == null ? "" : detectedComponentType,
                 "", "", "", false);
+
+        jsonApi.writeValue(rdtDetailsConfirmationPage, CovidConstants.FormFields.RDT_MANUFACTURER, deviceDefinitionProcessor.extractManufacturerName(deviceId), "", "", "", false);
+        jsonApi.writeValue(rdtDetailsConfirmationPage, CovidConstants.FormFields.RDT_DEVICE_NAME, deviceDefinitionProcessor.extractDeviceName(deviceId), "", "", "", false);
```

Comment:
manufacturer and device name are already extracted elsewhere in this method. They can be declared once and used here.

Reply:
Also add tests to verify that these values are written into the form.

Reply:
done